docs/release-notes/8.4.0-2.md

```diff
@@ -0,0 +1,92 @@
+
+* Debian (12)
+
+Percona XtraBackup 8.4.0-2 Pro build includes the [`Reduced backup lock time`](../reduction-in-locks.md) feature. With the addition of the [`--lock-ddl=REDUCED`](./xtrabackup-option-reference.md#lock-ddl) option this feature minimizes the time the server remains locked by xtrabackup during both full and incremental backups. Now, you can execute `Data Definition Language` (DDL) operations while the backup is in progress.
+
+### Benefits
+
+The `--lock-ddl=REDUCED` option features are as follows:
+
+* Acquire and release the backup lock quickly: The backup process minimizes the time it holds the backup lock, allowing for concurrent DDL operations.
+
+    ??? note "Comparing the backup duration with --lock-ddl=REDUCED"
+
+        While we strive to provide accurate performance benchmarks, real-world results may vary depending on your hardware and software configurations.
+
+        The following tables illustrate the approximate time differences between `--lock-ddl=ON` and `--lock-ddl=REDUCED` for local and cloud backups (Amazon S3). The `Improvement (X times)` column shows how many times less the server is locked by xtrabackup using  `--lock-ddl=REDUCED` compared to `--lock-ddl=ON`.
+
+        === "Local backup"
+
+	        | Data size in gigabytes | Backup duration with --lock-ddl=ON in seconds| Backup duration with --lock-ddl=REDUCED in seconds| Improvement (X times)  |
+            |-----------------------------|---------------|--------------------|--------------------------|
+            | 50 GB                       | 460.2 sec         | 2.169 sec             | 212.17                   |
+            | 100 GB                      | 901.8 sec        | 1.305 sec             | 691.03                   |
+            | 200 GB                      | 1820.4 sec       | 1.347 sec             | 1351.45                  |
+            | 400 GB                      | 3580.2 sec       | 1.24 sec             | 2887.26                  |
+            | 500 GB                      | 5436 sec         | 1.264 sec             | 4300.63                  |
+
+        === "Cloud (Amazon s3) backup"
+
+            | Data size in gigabytes | Backup duration with --lock-ddl=ON in seconds | Backup duration with --lock-ddl=REDUCED in seconds | Improvement (X times) |
+            |----------------------------|---------------|--------------------|--------------------------|
+            | 50 GB                      | 469.8 sec         | 3.859 sec             | 121.74                   |
+            | 100 GB                     | 927.6 sec         | 4.072 sec             | 227.80                   |
+            | 200 GB                     | 1851.6 sec        | 4.408 sec             | 420.05                   |
+            | 400 GB                     | 3888 sec         | 3.948 sec             | 984.80                   |
+            | 500 GB                     | 4896 sec         | 4.065 sec             | 1204.43                  |
```

Comment:
[optional] This can come before Limitations sections? It seems to be in middle of description of reduced lock feature

Reply:
I've added those tables to the benefits subsection, the same as in the Reduction in licks feature